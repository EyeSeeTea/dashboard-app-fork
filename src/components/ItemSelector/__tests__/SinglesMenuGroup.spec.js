import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { SinglesMenuGroup } from '../SinglesMenuGroup'

describe('SinglesMenuGroup', () => {
    const wrapper = props => shallow(<SinglesMenuGroup {...props} />)

    it('matches snapshot', () => {
        const props = {
            acAddDashboardItem: jest.fn(),
            category: {
                header: 'ponies',
                items: [
                    {
                        type: 'colorful',
                        name: 'Rainbow Dash',
                    },
                    {
                        type: 'greytone',
                        name: 'B&W',
                    },
                ],
            },
        }
        expect(toJson(wrapper(props))).toMatchSnapshot()
    })
})
