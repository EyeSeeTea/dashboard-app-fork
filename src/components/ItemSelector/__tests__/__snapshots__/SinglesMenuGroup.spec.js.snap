// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SinglesMenuGroup matches snapshot 1`] = `
<Fragment>
  <HeaderMenuItem
    title="ponies"
  />
  <ContentMenuItem
    key="colorful"
    name="Rainbow Dash"
    onInsert={[Function]}
    type="colorful"
  />
  <ContentMenuItem
    key="greytone"
    name="B&W"
    onInsert={[Function]}
    type="greytone"
  />
</Fragment>
`;
